### DISCUSSION TITLE
Showing the institute/assigned user for hidden observations

### DISCUSSION THREAD

**Discussion by northwestwitch** [Clinical-Genomics/scout#2412] [General]
A user was wondering why there are 7 observation but they see only 3 cases. That's because they are only allowed to see their cases and the cases from collaborating institutes.

Would it be allowed/informative to show the institute those hidden cases belong to, or even the user assigned to these cases, so they can contact them and ask regarding phenotype etc?

**Comment by dnil** [comment]
With us (cust002 and related) that would be acceptable, but I am a little worried what eg Immunology would say if a pathologist calls and asks about a cancer variant? Let’s bring that to a user meeting? We could also explicitly state the number of hits to other institutes, and that  according to previous agreement CG can broker the connection, minding what type of questions would be appropriate eg via a ticket?
